addon/array/uniq-by.js

```diff
@@ -12,23 +10,8 @@ export default createClassComputed(
       if (array === undefined || key === undefined) {
         return array;
       }
-      if (!array.uniqBy) {
-        // TODO: polyfill this
-        // from https://github.com/emberjs/ember.js/blob/v2.11.0/packages/ember-runtime/lib/mixins/enumerable.js#L1094-L1105
-        let ret = emberA();
-        let seen = Object.create(null);
 
-        array.forEach(item => {
-          let guid = guidFor(get(item, key));
-          if (!(guid in seen)) {
-            seen[guid] = true;
-            ret.push(item);
-          }
-        });
-
-        return ret;
-      }
```

Comment:
Maybe I don't get why this was necessary but `emberA(array)` should really have the same result.

Reply:
Nice! Simplify all the things :)

Reply:
Apparently `uniqBy` doesn't exist in Ember 1.13: https://travis-ci.org/kellyselden/ember-awesome-macros/jobs/270136987#L2742 😞

Reply:
#269 for reference